tibco_ems/README.md

```diff
@@ -10,15 +10,76 @@ Follow the instructions below to install and configure this check for an Agent r
 
 ### Installation
 
-The TIBCO EMS check is included in the [Datadog Agent][2] package.
-No additional installation is needed on your server.
+The TIBCO EMS check is included in the [Datadog Agent][2] package. No additional installation is needed on your server.
 
 ### Configuration
 
 1. Edit the `tibco_ems.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your TIBCO EMS performance data. See the [sample tibco_ems.d/conf.yaml][4] for all available configuration options.
 
 2. [Restart the Agent][5].
 
+#### Metric collection
+
+##### Create your Tibco EMS command script
+
+1. The Tibco EMS integration utilizes the `tibemsadmin` CLI tool provided by Tibco EMS. In order to reduce the number of calls to `$sys.admin` queue, we batch the queries we make to Tibco in the form of a script. This script path, as well the absolute path of the `tibemsadmin` binary, need to be passed to the integration configuration in order to collect your Tibco EMS metrics.
```

Comment:
You should mention some where in this doc that the dd-agent user will need to have execute permissions to the binary.